primitives/runtime/src/random_number_generator.rs

```diff
@@ -27,6 +27,8 @@ use crate::traits::{Hash, TrailingZeroInput};
 ///
 /// It can be saved and later reloaded using the Codec traits.
 ///
+/// (It is recommended to use the `rand_chacha` crate as an alternative to this where possible.)
+///
```

Comment:
Where does this come from?

Reply:
https://github.com/paritytech/substrate/issues/7444#issuecomment-726063259